addons/board/static/tests/dashboard_tests.js

```diff
@@ -88,7 +88,7 @@ odoo.define("board.dashboard_tests", function (require) {
                 {
                     Component: AddToBoard,
                     groupNumber: 4,
-                    isDisplayed: ({ config }) => config.actionType === "ir.actions.act_window",
+                    isDisplayed: ({ config }) => config.actionType === "ir.actions.act_window" && config.actionId,
```

Comment:
this is the line you're testing... You should export the `addToBoardItem`, import it here and add it to the registry.